app/ClientLayout.tsx

```diff
@@ -5,24 +5,26 @@ import React, { useRef } from "react"
 import { Hero } from "@/components/Hero"
 import { HeroCard } from "@/components/HeroCard"
 import { ImpactBanner } from "@/components/ImpactBanner"
-import { FeaturedProjects } from "@/components/FeaturedProjects"
+import { FeaturedCarousel } from "@/components/FeaturedCarousel"
 
 export default function ClientLayout() {
-  const featuredProjectsRef = useRef<HTMLDivElement>(null)
+  const featuredCarouselRef = useRef<HTMLDivElement>(null)
 
-  const scrollToFeaturedProjects = () => {
-    featuredProjectsRef.current?.scrollIntoView({ behavior: 'smooth' })
+  const scrollToFeaturedCarousel = () => {
+    featuredCarouselRef.current?.scrollIntoView({ behavior: 'smooth' })
   }
 
   return (
     <>
       <div className="relative w-full">
-        <Hero onViewEventsClick={scrollToFeaturedProjects} />
-        <HeroCard />
-        <ImpactBanner />
-      </div>
-      <div ref={featuredProjectsRef} className="mt-20 w-full">
-        <FeaturedProjects />
+          <div className="flex flex-col space-y-24"> 
+            <Hero onViewEventsClick={scrollToFeaturedCarousel} />
```

Comment:
So when I click the "View Events" button this is scrolling to the Featured Projects Section.

Change the scrollTo to go to the events ref?

Reply:
Ashley and I noticed this yesterday. We'll make an issue and address it in a refactor.

Reply:
updated this to the events ref, but this scroll behavior thing needs refactoring in a separate PR. also, duplicate section header components need to be addressed in the Events PR!